en/code_editor/README.md

```diff
@@ -23,3 +23,16 @@ Sublime Text is a very popular editor with a free evaluation period. It's easy t
 Atom is an extremely new code editor created by [GitHub](http://github.com/). It's free, open-source, easy to install and easy to use. It's available for Windows, OSX and Linux.
 
 [Download it here](https://atom.io/)
+
+
+
+# Why are we installing a code editor?
+
+You might be wondering why we are installing this special code editor software, rather than using something like Word or Notepad.
+
+The first is that code needs to be **plain text**, and the problem with programs like Word and Textedit is that they don't actually produce plain text, they produce rich text (with fonts and formatting), using custom formats like rtf.
+
+The second reason is that code editors are specialised in editing code, so they can provide helpful features, like syntax highlighting, or automatically closing quotes for you.
```

Comment:
Isn't "syntax highlighting" defined only in a later chapter?

I would either drop it or at least explain what that means in a few words.